### PR TITLE
fix aws metadata util script

### PR DESCRIPTION
**Pull request by yuhuyoyo**
should not echo logs in get_metadata_value

### CODE REVIEW

File: startupscript/butane/aws/metadata-utils.sh [hunk 1]
```diff
@@ -4,24 +4,29 @@
 # to retrieve or modify instance tags. It is run on the VM host. AWS CLI is not installed on Flatcar VM by default so
 # we are running the AWS CLI in a container.
 
-# Retrieves an instance tag set on the VM.
+# Retrieves an instance tag set on the VM. If the tag is not set, it returns 0.
```
Comment (mbookman) [inline startupscript/butane/aws/metadata-utils.sh:7]: Does returning zero generalize for all such tags?
Why not just return an empty string? That feels more natural.
Alternatively, the function could take an optional second argument which would be the default value to return.

```
MY_STRING="$(get_metadata_value "my-string-value" "")"
MY_NUMBER="$(get_metadata_value "my-numeric-value" "0")"
```

Comment (yuhuyoyo) [reply startupscript/butane/aws/metadata-utils.sh:7]: right i got tunnel visioned.. echo empty string instead

Comment (mbookman) [inline startupscript/butane/aws/metadata-utils.sh:7]:
```suggestion
# Retrieves an instance tag set on the VM. If the tag is not set, it returns an empty string..
```

**Review by mbookman — APPROVED** [review]
One small thing nit. Otherwise LGTM.

**Pull request merged**